intf/vernacexpr.ml

```diff
@@ -317,7 +317,7 @@ type cumulative_inductive_parsing_flag =
 type vernac_expr =
   (* Control *)
   | VernacLoad of verbose_flag * string
-  | VernacTime of vernac_expr located
+  | VernacTime of bool * vernac_expr located (* true if batch time mode *)
```

Comment:
Can you write "`-time` argument" instead of "batch"? It would have helped me to understand what you meant.